enhancements/installer/ibm-cloud-for-power-vs-ipi.md

```diff
@@ -0,0 +1,278 @@
+4. [Upload/Create<sup>*</sup>](#coreos-ova-considerations) the required boot images.
+5. Create VM by specifying configurable values:
+    - \# of CPU / Cores
+    - Shared vs. Dedicated CPUs
+    - Server class (s922 vs. e980)
+    - Machine type
+    - \# of Memory
+    - Boot Image
+    - Additional disk volume(s)
+    - Storage Type (Tier 1 or 3)
+    - Network (public, private)
+
+*Note*: Disk size of boot image == size of boot image
+
+[Power VS Reference Doc][power-vs-reference-doc]
+
+### Open Questions
+
+#### OVA Import Time
+Is 5 minutes for important a target deployment image satisfactory for an IPI deployment. Additonally, is it even
+possible for the Power VS team to reduce the current import time from 30 minutes to 5 (for the purposes of meeting this
+requirement)?
+
+#### Deployment Time
+What is the maximum allowable time for a successful deployment to take. Is under an hour sufficient?
+
+#### DHCP in the IBM Cloud
+How soon will we have DHCP in the Power VS platform. If we can't get DHCP, do we have other options?
+
+#### Flavors or Granular Control
+One thing that is being discussed is whether it would be better to expose a subset of cloud instance "flavors" with
+pre-defined compute, storage, CPU cores, etc., or to have the installer mirror the current instance-creation requirement
+for Power VS.
+
+One thing that is clear is that limiting the number of installation options is key to long term sustainability.
+
+### Test Plan
+
+Multi-Arch currently tests results by running upgrade and e2e tests with the OpenShift Prow CI environment. Our test
+plan would extend this to include these IPI scenarios.
+
+### Graduation Criteria
+
+- Our initial target is to work towards a `Dev Preview`, then graduate towards future milestones. This is expected
+to take a total of three releases, but could be more or less depending on how quickly we meet the requirements
+enumerated in the graduation criteria.
+- `Dev Preview` will graduate to  `Tech Preview` and `Tech Preview` to `GA` in OpenShift
+- [Deprecation policy][deprecation-policy]
+
+[deprecation-policy]: https://kubernetes.io/docs/reference/using-api/deprecation-policy/
+
+#### Dev Preview Requirements
+- [ ] Implemented APIs necessary for IPI installation
+- [ ] DHCP requirement solved by the Power VS team
+- [ ] Builds of CoreOS are available in the IBM Cloud for installation
+- [ ] First pass at defining which installation criteria belong in the installer and which are day 2 configurations
+- [ ] Developer documentation
+- [ ] Plan defined to get feedback from developers
+- [ ] Sufficient time for feedback (6 weeks)
+- [ ] Tile up
+
+#### Dev Preview -> Tech Preview
+
+- [ ] Full functionality / ability to utilize the enhancement end to end
+- [ ] Full installation takes at most an hour
+- [ ] Stabilize API in terms of configuration options for install and day 2
+- [ ] End user documentation
+- [ ] CI e2e jobs created to cover installation against this platform
```

Comment:
For this, are you envisioning a hybird CI approach where we use the existing prow setup and call out to Power VS? And then for the GA, our Power VS IPI CI is entirely on the prow cluster living in Power VS? If so we should discuss with the CI team.

Reply:
I've tried to answer this in a new "CI Plan" section. Will want feedback from @deepsm007 and the CI team once I push this draft live.